lib/TH/generic/THTensorLapack.c

```diff
@@ -513,7 +513,10 @@ void THTensor_(potrf)(THTensor *ra_, THTensor *a, const char *uplo)
 
   /* Run Factorization */
   THLapack_(potrf)(uplo[0], n, THTensor_(data)(ra__), lda, &info);
-  THLapackCheck("Lapack Error %s : A(%d,%d) is 0, A cannot be factorized", "potrf", info, info);
+  THLapackCheckWithHandler(
```

Comment:
There are at least three similar call sites of the erstwhile `THLapackCheck` that may require the same cleanup. They just don't have test coverage :-)